README.md

```diff
@@ -25,4 +25,4 @@ Currently, the library only implements a subset of the permissable CEF
 extensions that were personally useful. I'm very happy to take PRs to extend
 coverage to the fully valid CEF extension set.
 
-[CEF]: https://www.protect724.hpe.com/servlet/JiveServlet/downloadBody/1072-102-9-20354/CommonEventFormatv23.pdf
+[CEF]: https://web.archive.org/web/20190926101816/https://community.microfocus.com/dcvta86296/attachments/dcvta86296/connector-documentation/1197/2/CommonEventFormatV25.pdf
```

Comment:
```suggestion
[CEF]: https://web.archive.org/web/20190930102530/https://community.microfocus.com/dcvta86296/attachments/dcvta86296/connector-documentation/1197/2/CommonEventFormatV25.pdf
```

weirdly that old archive link doesn't work anymore

Reply:
I hope it is now saved by the Wayback Machine.
Do you think, attaching this guide to the project would be useful? Or it's not a good idea?